pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -129,22 +129,19 @@ func getConfigurationName(args []string) (string, error) {
 
 //  Get the configurator based on the ipfailover type.
 func getPlugin(name string, f *clientcmd.Factory, options *ipfailover.IPFailoverConfigCmdOptions) (ipfailover.IPFailoverConfiguratorPlugin, error) {
-	//  Currently, the only supported plugin is keepalived (default).
-	plugin, err := keepalived.NewIPFailoverConfiguratorPlugin(name, f, options)
+	var plugin ipfailover.IPFailoverConfiguratorPlugin
+	var err error
 
-	switch options.Type {
-	case ipfailover.DefaultType:
-		//  Default.
-	// case <new-type>:  plugin, err = makeNewTypePlugin()
-	default:
-		return nil, fmt.Errorf("No plugins available to handle type %q", options.Type)
-	}
+	if options.Type == ipfailover.DefaultType {
+		plugin, err = keepalived.NewIPFailoverConfiguratorPlugin(name, f, options)
```

Comment:
You don't need the `plugin` and `err` variables defined on line 132 and 133, you could just scope it to be local and use the `:=` operator here (as `plugin` and `err` are just used locally in the `if` block.